provd/rest/api/api.yml

```
@@ -47,6 +47,84 @@ paths:
           schema:
             $ref: '#/definitions/GeneralConfigsObject'
 
+  /configure/plugin_server:
+    get:
+      summary: Get the configuration's plugin_server
+      description: |
+        **Required ACL:** `provd.configure.read`
```

```suggestion
      description: '**Required ACL:** `provd.configure.read`'
```
Avoid pipe when block text is not used